### ISSUE TITLE
`plot.emmGrid()` lacks backwards compatibility with `intervals`

### ISSUE THREAD

**Issue by gprochilo** [rvlenth/emmeans#137]
Hi,

The latest update breaks backwards compatibility when plotting a `summary_emm`. This includes:

1. Confidence intervals cannot be turned off using the `intervals` argument: 

```{r eval = FALSE}
emmeans::pigs %>% 
lm(log(conc) ~ source + factor(percent), data = .) %>% 
emmeans(., "source") %>% 
pairs() %>% 
summary(., infer = c(T, T), level = 0.95, null = 0) %>% 
plot(intervals = FALSE, colors = "black")
```
2. Plotting a `summary_emm` requires that the user specify the new `colors` argument. If no argument is specified it throws an error. 

```{r eval = FALSE}
> emmeans::pigs %>% 
+   lm(log(conc) ~ source + factor(percent), data = .) %>% 
+   emmeans(., "source") %>% 
+   pairs() %>% 
+   summary(., infer = c(T, T), level = 0.95, null = 0) %>% 
+   plot(intervals = FALSE)
Error in .plot.srg(x, y, horizontal, xlab, ylab, layout, ...) : "colors" is missing, with no default
```

Perhaps it would be better to set a default `colors` value?

Cheers,

Guy

**Comment by rvlenth** [comment]
Ouch! Thanks for catching this. I failed to test the modified code against `summary_emm` objects.

I think I have fixed it. Your example:
```r
emmeans::pigs %>% 
    lm(log(conc) ~ source + factor(percent), data = .) %>% 
    emmeans(., "source") %>% 
    pairs() %>% 
    summary(., infer = c(T, T), level = 0.95, null = 0) -> 
    smry
```
```r
plot(smry)
```
![image](https://user-images.githubusercontent.com/26777519/64477338-f12f0680-d15f-11e9-8bc1-07ddfebae93a.png)

```r
plot(smry, intervals = FALSE)
```
![image](https://user-images.githubusercontent.com/26777519/64477344-07d55d80-d160-11e9-9a2c-232c785f2ee9.png)

Note: I also clarified the help page -- especially that `PIs` is not available with `emm_summary` objects.

The repaired version will be available with the next push. Thanks again for ereporting this.
